docs/guide/backup.md

```diff
@@ -22,7 +22,11 @@ var/logs/*
 3\. Create a dump of the database:
  
 ```
-mysqldump -u root --add-drop-table example > db_backup.sql
+# MySQL
+mysqldump -u root --add-drop-table <database_name> > db_backup.sql
```

Comment:
Let's also change "root" to `<database_user>`. If a production setup uses root user it's rather bad, so unlikely to happen.
```suggestion
mysqldump -u <database_user> --add-drop-table <database_name> > db_backup.sql
```